jinahub/encoders/image/TimmImageEncoder/tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pillow==8.2
+pytest==6.2.4
+pytest-mock==3.6.1
+pytest-lazy-fixture==0.6.3
```

Comment:
We also don't need them I think?

Reply:
pytest-lazy-fixture is used in unit test test_encode_image_returns_correct_length